dap_job_quality/pipeline/prodigy/sentence_classifier_recipe.py

```diff
@@ -0,0 +1,157 @@
+"""
+Usage:
+
+To start the app:
+```
```

Comment:
Suggest making it clear you have to be in the prodigy directory to run.
```suggestion
cd dap_job_quality/pipeline/prodigy
```